.vib/apache/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/apache"
         },
-        "runtime_parameters": "InNlcnZpY2UiOgogICJwb3J0IjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "ImNvbnRhaW5lclBvcnRzIjoKICAiaHR0cCI6IDgwODAKICAiaHR0cHMiOiA4NDQzCiJzZXJ2aWNlIjoKICAicG9ydCI6IDgwCiAgInR5cGUiOiAiTG9hZEJhbGFuY2VyIgoiaG9zdEFsaWFzZXMiOgogIC0gImlwIjogMTI3LjAuMC4xCiAgICAiaG9zdG5hbWVzIjoKICAgICAgICAtICJzdGF0dXMubG9jYWxob3N0Ig==",
```

Comment:
These values are incorrect: the path to the port of the service are do not follow the same structure as in `values.yaml`

https://github.com/bitnami/charts/blob/75336a70b8efb5581c012b9bf9fa7b4d5535badf/bitnami/apache/values.yaml#L408-L410

Additionally, you should also set the value for `https` as we will be using it in the tests.

Reply:
You can also drop the `containerPorts` parameters here.